cql3/expr/expression.cc

```diff
@@ -2323,5 +2323,31 @@ bool is_single_column_restriction(const expression& e) {
 
     return result;
 }
+
+const column_value& get_the_only_column(const expression& e) {
+    if (find_in_expression<unresolved_identifier>(e, [](const auto&) {return true;})) {
+        on_internal_error(expr_logger, format("get_the_only_column - expression is not prepared: {}", e));
```

Comment:
I guess the role of the function is to convert a restriction to a computation that gets a key column?